lib/facebook_ads/configuration.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+module FacebookAds
+  class Configuration
+    def initialize
+      @config = {}
+    end
+
+    def configured?
```

Comment:
present? or valid? would read much nicer in my opinion.
`raise NotConfiguredError unless configuration.configured?`
vs
`raise NotConfiguredError unless configuration.present?`